pkg/gardenlet/features/features.go

```diff
@@ -28,9 +28,9 @@ var (
 		features.Logging:                {Default: false, PreRelease: featuregate.Alpha},
 		features.HVPA:                   {Default: false, PreRelease: featuregate.Alpha},
 		features.HVPAForShootedSeed:     {Default: false, PreRelease: featuregate.Alpha},
-		features.ManagedIstio:           {Default: false, PreRelease: featuregate.Alpha},
+		features.ManagedIstio:           {Default: false, PreRelease: featuregate.Beta},
 		features.KonnectivityTunnel:     {Default: false, PreRelease: featuregate.Alpha},
-		features.APIServerSNI:           {Default: false, PreRelease: featuregate.Alpha},
+		features.APIServerSNI:           {Default: false, PreRelease: featuregate.Beta},
```

Comment:
I did not enable them by default yet. Should we do it?

Reply:
Yes.

Reply:
Also, enable them by default here https://github.com/gardener/gardener/blob/5eefe2db2e64db1d7947c3bdf605ac7e7879346b/example/20-componentconfig-gardenlet.yaml#L88-L90